modules/vba_documents/app/sidekiq/vba_documents/upload_scanner.rb

```diff
@@ -55,6 +57,10 @@ def bucket
         s3.bucket(Settings.vba_documents.s3.bucket)
       end
     end
+
+    def log_details(upload)
+      { 'job' => self.class.name }.merge(upload.as_json)
+    end
```

Comment:
Need to always get the freshest version of `upload` since it's constantly changing inside this job. Want to avoid the log data being out of date.

Reply:
# 🧠